src/Sentry/Internal/BackgroundWorker.cs

```diff
@@ -274,6 +274,10 @@ public void Dispose()
                 // If the queue is empty it will quit immediately
                 WorkerTask.Wait(_options.ShutdownTimeout);
             }
+            catch (OperationCanceledException)
+            {
+                // Ignored
```

Comment:
What about we just debug log this?

```suggestion
                 _options.DiagnosticLogger?.LogDebug("Stopping the background worker due to a cancellation");
```